Library/Homebrew/cask/artifact/pkg.rb

```diff
@@ -65,7 +65,8 @@ def run_installer(command: nil, verbose: false, **_options)
             "USER"     => User.current,
             "USERNAME" => User.current,
           }
-          command.run!("/usr/sbin/installer", sudo: true, args: args, print_stdout: true, env: env)
+          sudo = target != "CurrentUserHomeDirectory"
```

Comment:
Should we expect the cask to pass paths as strings, but special values like `CurrentUserHomeDirectory` as a `Symbol`?
```suggestion
          sudo = target != :CurrentUserHomeDirectory
```